tracer/build/_build/Build.GitHub.cs

```diff
@@ -94,6 +99,92 @@ await client.Issue.Update(
             Console.WriteLine($"PR assigned");
         });
 
+    Target LLMReport => _ => _
+           .Unlisted()
+           .Requires(() => GitHubRepositoryName)
+           .Requires(() => GitHubToken)
+           .Requires(() => OpenAIKey)
+           .Requires(() => PullRequestNumber)
+           .Executes(async () =>
+           {
+               await GenerateLLMReport(false);
+           });
+
+    Target LLMLocalReport => _ => _
+       .Unlisted()
+       .Requires(() => GitHubRepositoryName)
+       .Requires(() => GitHubToken)
+       .Requires(() => OpenAIKey)
+       .Requires(() => PullRequestNumber)
+       .Executes(async () =>
+       {
+           await GenerateLLMReport(true);
+       });
+
+    private async Task GenerateLLMReport(bool executeLocal = true)
+    {
```

Comment:
I'm trying to _reduce_ the number of targets 😅 Could you consider merging these and using the existing `IsLocalBuild` flag instead? That way we can also remove the `GenerateLLMReport` function entirely too:


```suggestion
    Target LLMReport => _ => _
           .Unlisted()
           .Requires(() => GitHubRepositoryName)
           .Requires(() => GitHubToken)
           .Requires(() => OpenAIKey)
           .Requires(() => PullRequestNumber)
           .Executes(async () =>
           {
               var executeLocal = IsLocalBuild;
               // ..etc
```